massa-execution-worker/src/execution.rs

```diff
@@ -616,6 +620,59 @@ impl ExecutionState {
         Ok(())
     }
 
+    pub fn execute_denunciation_op(
```

Comment:
it feels strange to charge the operation fee for the denunciation. We can discuss this